auth0_flutter/lib/src/web/auth0_flutter_plugin_real.dart

```diff
@@ -35,7 +32,13 @@ class Auth0FlutterPlugin extends Auth0FlutterWebPlatform {
   @override
   Future<void> loginWithRedirect(final LoginOptions? options) {
     final authParams = _stripNulls(AuthorizationParams(
-        audience: options?.audience, redirect_uri: options?.redirectUrl));
+        audience: options?.audience,
+        redirect_uri: options?.redirectUrl,
+        organization: options?.organizationId,
+        invitation: options?.invitationUrl,
+        max_age: options?.idTokenValidationConfig?.maxAge,
```

Comment:
This is a little bit weird, I think we'll have to rethink how this is done. In the SDK client, for some reason `max_age` is dealt with separately from `leeway` and `issuer`, which are both specified at initialization time.